src/nautilus_launch/launch/surface.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
   <group ns="surface_control">
-    <node name="server" pkg="uwrov_server" type="main_server.py" output="screen"/>
+    <!--<node name="server" pkg="uwrov_server" type="main_server.py" output="screen"/>-->
     <node name="interface" pkg="uwrov_interface" type="launch_interface.bash"/>
```

Comment:
Is there a significance to running the interface without the server backend?

Reply:
Currently there is a bug where the server and the robot_module shouldn't run at the same time. Going to resolve this issue next patch.